cmd/harborAPI.go

```diff
@@ -305,3 +325,43 @@ func UpdateContainerImage(token string, shipment string, composeShipment Compose
 	//call api
 	update(token, url, payload)
 }
+
+// SaveNewShipmentEnvironment bulk saves a new shipment/environment
+func SaveNewShipmentEnvironment(shipment NewShipmentEnvironment, token string) bool {
+
+	//POST /api/v1/shipments
+	res, body, err := create(token, harborURI+"/api/v1/shipments", shipment)
+
+	if err != nil || res.StatusCode != 200 {
+		fmt.Printf("creating shipment was not successful: %v \n", body)
+		return false
+	}
+
+	//api returns an object with an errors property that is
+	//false when there are no errors and an object if there are
+	if !strings.Contains(body, "errors\": false") {
+		return false
```

Comment:
So after looking at this more closely, if there's an error, it's currently writing the HTTP response body to the console.  I'm not entirely sure the shape and meaning of the error data coming back so I'm thinking we just leave as-is for now, until we move to the new server API.